SAMtags.tex

```diff
@@ -62,6 +62,7 @@ \section{Standard tags}
   {\tt CC} & Z & Reference name of the next hit \\
   {\tt CM} & i & Edit distance between the color sequence and the color reference (see also {\tt NM}) \\
   {\tt CO} & Z & Free-text comments \\
+  {\tt CR} & Z & Cellular barcode sequence as reported by the sequencer (uncorrected) with the respective quality scores stored in CY\\
```

Comment:
Change `in CY\\` to `in the {\tt CY} tag`.

Reply:
How about this:
`Sequence bases of the cellular barcode bases (uncorrected) with the respective quality scores stored in the {\tt CY} tag`?  I am trying to match the style of writing in the `RX` tag